modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -240,4 +245,9 @@ $ alias coreos-installer='podman run --privileged --pull always --rm \
 ----
 $ coreos-installer iso ignition embed -fi sno/bootstrap-in-place-for-live-iso.ign fcos-live.iso
 ----
-endif::openshift-origin[]
\ No newline at end of file
+endif::openshift-origin[]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
No links in modules. Add to the assembly, which had an _Additional resources_ section. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/b7ad25b7-e041-4f69-ab35-730342e9831b)

Reply:
Updated As Suggested.